### ISSUE TITLE
Celebrity

### ISSUE THREAD

**Issue by uikop888** [uikop888/Building-program#3]
Celebrities are similar to Us, they flip to dating apps to discover romance and begin relationships. Stars which include Eric Stonestreet and Chelsea Handler had been open approximately their use of courting apps, which includes Tinder and Bumble, however even extra well-known oldsters have opted to apply Raya as you can see the detail [about nav](https://wealthysquad.com/nav-net-worth/).

**Comment by durefishan00991188** [comment]
Yes, exactly it is same in our country. Our celebrities durefishan saleem is doing same.
